open-craft/edx-simple-theme
# Initial version of branding package [BB-4092]

This PR updates this theme to also work as an Open edX branding package. 

Branding packages are a concept introduced by [OEP-0048](https://open-edx-proposals.readthedocs.io/en/latest/oep-0048-brand-customization.html) that create a standard way to customise the theme of MFEs. 

It refactors the existing styling into scss common between edx-platform and MFEs which is then imported by the platform theme and the branding package. It also imports the variables and overrides added by the [simple_theme role in edx/configuration](https://github.com/edx/configuration/tree/afffecc7aaf38bd018c39fffcc5190da783f1c17/playbooks/roles/simple_theme) so that existing mechanisms for customising theming will continue working. 

**JIRA tickets**: https://gitlab.com/opencraft/dev/branding-and-theming/-/issues/6

**Sandbox URL**: https://branding-theming-sandbox.opencraft.hosting/


**Testing instructions**:

1. Create an `_lms_overrides.scss` file with any scss overrides. 
2. Create a `comon-variables.scss` file with the variables you'd like to override.
3. Install this as a regular Open edX theme. 
4. Install this as a branding package as described [here](https://open-edx-proposals.readthedocs.io/en/latest/oep-0048-brand-customization.html#decision)
5. Compile static assets for the LMS,
6. Make sure the MFE packages are up to date and restart the dev server. 
7. The theme colours and overrides should be applied to both edx-platform and the MFE. 


**Reviewers**
